src/CSSBattle/10_CloakedSpirits/Yongbin.html

```diff
@@ -0,0 +1,109 @@
+<div class="col1"></div>
+<div class="col2"></div>
+<div class="col3"></div>
```

Comment:
Consider naming the classes `colLeft` `colMid` `colRight` to make it even clearer for the reader.

Reply:
If you are planning on renaming, you can use VSCode's `Rename All Occurrences` to help you rename all the same-name variables together!

Reply:
Resolved in commit 5ec733b.